wg-automl/charter.md

```diff
@@ -19,7 +19,7 @@ The WG covers researching, developing and operating various targets of ML automa
 #### Cross-cutting and Externally Facing Processes
 
 - Coordinating with Training WG to make sure that all distributed training jobs can be used in AutoML experiments.
-- Coordinating with Control Plane WG to ensure that AutoML manifests are properly deployed with Kubeflow.
+- Coordinating with Manifests WG to ensure that AutoML manifests are properly deployed with Kubeflow.
```

Comment:
> Coordinating with Manifests WG to ensure that AutoML manifests are properly deployed with Kubeflow.

Why would this be necessary?

Reply:
> Why would this be necessary?

@jlewi those are not new scopes, but pre-existing ones.
I simply amended the name since WG Control Plane doesn't exist anymore.